speechbrain/lobes/models/dual_path.py

```diff
@@ -253,12 +253,29 @@ def forward(self, x):
                        N = number of filters
                        L = time points
         """
-
-        if x.dim() not in [2, 3]:
-            raise RuntimeError(
-                "{} accept 3/4D tensor as input".format(self.__name__)
-            )
-        x = super().forward(x if x.dim() == 3 else torch.unsqueeze(x, 1))
+        # Ensure scriptability
+        # if x.dim() not in [2, 3]:
+        #     raise RuntimeError(
+        #         "{} accept 3/4D tensor as input".format(self.__name__)
+        #     )
+        # x = super().forward(x if x.dim() == 3 else torch.unsqueeze(x, 1))
+        # is not scriptable
+
+        if x.dim() != 3:
+            x = torch.unsqueeze(x, 1)
+        output_padding = self._output_padding(
+            x, None, self.stride, self.padding, self.kernel_size, self.dilation
+        )
+        x = F.conv_transpose1d(
```

Comment:
It looks like this is adding convolution step that wasn't there before. Is that correct?

Reply:
It was there. Inside `super().forward(...)` call.

In this module I just have removed exeption raising and replace `super().forward(...)` call by copy of this code https://github.com/pytorch/pytorch/blob/36449ea93134574c2a22b87baad3de0bf8d64d42/torch/nn/modules/conv.py#L770 . Because both operations are not JITable.